src/svg-renderer.js

```diff
@@ -292,6 +294,25 @@ class SvgRenderer {
         return largestStrokeWidth;
     }
 
+    /**
+     * Find all instances of a URL-referenced `stroke` in the svg. In 2.0, all gradient strokes
+     * have a round `stroke-linejoin` and `stroke-linecap`... for some reason.
+     */
+    _setGradientStrokeRoundedness () {
+        const elements = this._collectElements();
+
+        for (const elt of elements) {
+            if (elt.getAttribute) {
```

Comment:
Does it make sense to add this filter to _collectElements?

Reply:
Done